tests/e2e/specs/SequenceFlows.spec.js

```diff
@@ -233,4 +234,23 @@ describe('Sequence Flows', () => {
 
     cy.get('.main-paper [data-type="standard.Link"] [joint-selector="line"]').should('have.attr', 'd', 'M 308 326 L 308 450');
   });
+
+  it('retains original background color when it can and cannot connect to an element', () => {
```

Comment:
Is this test doing two things? If so how do we know what has failed when it fails? I would prefer to see this broken out into separate tests.